### DISCUSSION TITLE
[GeoMechanicsApplication] Q2Q1 displacement - pressure element

### DISCUSSION THREAD

**Discussion by muhammedfurkanyilmaz** [KratosMultiphysics/Kratos#12072] [Q&A] [answered]
Dear all,

I'm modeling a dynamic consolidation problem with 2D plane strain elements. I'd like to use 8 noded quadrilateral elements for displacements and 4 nodes for pressure calculations. How can I do that? 
Any help will be appreciated. 

Regards
MFY

**Comment by loumalouomega** [comment]
@KratosMultiphysics/geomechanics FYI

Huuum, which element are you using or planing to use?, because I am not sure of understand what are you trying to do

**Reply by muhammedfurkanyilmaz** [reply]
I have already modeled that with UPwSmallStrainElement(2D4N) elements but I decided to redo that with the above-mentioned idea. Maybe the same elements could be used but I'm not sure.

**Reply by loumalouomega** [reply]
So you are already using mixed formulation elements. I think you cannot do what you want to do, @KratosMultiphysics/geomechanics ?

**Reply by muhammedfurkanyilmaz** [reply]
Hmm, thanks for your response.

**Reply by rubenzorrilla** [reply]
As far as I understand, you want to use a Q2/Q1 element. Up to my knowledge, this formulation is not implemented.

EDIT: let me change the title as it is quite misleading (Kratos supports using different elements in the same analysis by construction).

**Reply by avdg81** [reply, marked as answer]
Hi @muhammedfurkanyilmaz, many thanks for your follow-up question. If we (the team behind @KratosMultiphysics/geomechanics) understand your question correctly, you are looking for an 8-noded U-Pw small strain element with quadratic shape functions for the displacement field and linear shape functions for the water pressure field. If yes, then we suggest you to use the `"SmallStrainUPwDiffOrderElement2D8N"` elements, which is targeted at this use case. If no, could you please try to explain in different words what you are trying to achieve? Thanks.

Kind regards,
Anne

**Reply by muhammedfurkanyilmaz** [reply]
Hi Anne. 

''8-noded U-Pw small strain element with quadratic shape functions for the displacement field and linear shape functions for the water pressure field.'' that is exactly what I need. Thanks.

Regards,
MFY

**Reply by avdg81** [reply]
That's great Muhammed. Thanks for confirming. I will mark it as answer.

May we kindly ask you to mention us (@KratosMultiphysics/geomechanics) when you open new questions? In that way, we are directly notified and we will try to support you to the best of our ability. Thank you.

Also, may we ask you at which university you are conducting your PhD work?

**Reply by muhammedfurkanyilmaz** [reply]
Sure, It'll be better. Thanks for your attention. I'm conducting my PhD at Istanbul Technical University.
